ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/JdbcLedgerDao.scala

```diff
@@ -141,6 +147,141 @@ private class JdbcLedgerDao(
+
+  private val configurationEntryParser: RowParser[(Long, ConfigurationEntry)] =
+    (long("ledger_offset") ~
+      str("typ") ~
+      str("submission_id") ~
+      str("participant_id") ~
+      str("rejection_reason")(emptyStringToNullColumn).? ~
+      byteArray("configuration"))
+      .map(flatten)
+      .map {
+        case (offset, typ, submissionId, participantIdRaw, rejectionReason, configBytes) =>
+          val config = Configuration
+            .decode(configBytes)
+            .fold(err => sys.error(s"Failed to decode configuration: $err"), identity)
+          val participantId = LedgerString
+            .fromString(participantIdRaw)
+            .fold(
+              err => sys.error(s"Failed to decode participant id in configuration entry: $err"),
+              identity)
+
+          offset ->
+            (typ match {
+              case "accept" =>
+                ConfigurationEntry.Accepted(
+                  submissionId = submissionId,
+                  participantId = participantId,
+                  configuration = config
+                )
+              case "reject" =>
+                ConfigurationEntry.Rejected(
+                  submissionId = submissionId,
+                  participantId = participantId,
+                  rejectionReason = rejectionReason.getOrElse("<missing reason>"),
+                  proposedConfiguration = config
+                )
+
+              case _ =>
+                sys.error(s"getConfigurationEntries: Unknown configuration entry type: $typ")
+            })
+      }
+
+  override def getConfigurationEntries(
+      startInclusive: Long,
+      endExclusive: Long): Source[(Long, ConfigurationEntry), NotUsed] =
+    paginatingStream(
+      startInclusive,
+      endExclusive,
+      PageSize,
+      (startI, endE) => {
+        dbDispatcher.executeSql(s"load configuration entries [$startI, $endE[") { implicit conn =>
+          SQL_GET_CONFIGURATION_ENTRIES
+            .on("startInclusive" -> startI, "endExclusive" -> endE)
+            .as(configurationEntryParser.*)
+        }
+      }
+    ).flatMapConcat(Source(_))
+
+  private val SQL_INSERT_CONFIGURATION_ENTRY =
+    SQL(
+      """insert into configuration_entries(ledger_offset, recorded_at, submission_id, participant_id, typ, rejection_reason, configuration)
+        |values({ledger_offset}, {recorded_at}, {submission_id}, {participant_id}, {typ}, {rejection_reason}, {configuration})
+        |""".stripMargin)
+
+  override def storeConfigurationEntry(
+      offset: LedgerOffset,
+      newLedgerEnd: LedgerOffset,
+      externalOffset: Option[ExternalOffset],
+      recordedAt: Instant,
+      submissionId: String,
+      participantId: ParticipantId,
+      configuration: Configuration,
+      rejectionReason: Option[String]
+  ): Future[PersistenceResponse] = {
+    dbDispatcher.executeSql(s"store configuration entry submissionId=$submissionId") {
```

Comment:
The description is used for a metric and shouldn't contain spaces or data specific to a request (e.g. submissionId).
```suggestion
    dbDispatcher.executeSql("store_configuration_entry", Some("submissionId=$submissionId")) {
```